CONTRIBUTING.md

```diff
@@ -15,11 +15,11 @@ Follow either of the two links above to access the appropriate CLA and instructi
 
 ## Composition of This Repository and Where/How to Contribute
 
-The repository of the Kubernetes Python client consists of this main repository and a submodule, the [python-base](https://github.com/kubernetes-client/python-base) repository. The main repository contains mostly files that are generated by the OpenAPI generator from [this OpenAPI spec](scripts/swagger.json). The base repo is the utility part of the python client and allows developers to create their own kubernetes clients.
+The Kubernetes Python client consists of this main repository (repo) and the base repo, the [python-base](https://github.com/kubernetes-client/python-base) repository. The main repository contains mostly files that are generated by the OpenAPI generator from [this OpenAPI spec](scripts/swagger.json). The base repo is the utility part of the python client and allows developers to create their own kubernetes clients. The base repo is a submodule of the main repo.
```

Comment:
I'm assuming the part where it says "(repo)" should be a link? When I preview the file it's not a link.

Reply:
I'm actually saying that repo is a shorthand for repository.